lib/claim/token-holder-revenue-fund-contracts-ensemble.js

```diff
@@ -0,0 +1,359 @@
+'use strict';
+
+const {utils: {bigNumberify}} = require('ethers');
+const TokenHolderRevenueFundContract = require('./token-holder-revenue-fund-contract');
+const NestedError = require('../nested-error');
+
+const _ensemble = new WeakMap();
+const _spannedByCurrency = new WeakMap();
+const _firstAccrualOffset = new WeakMap();
+
+const currencyToKey = (currency) => `${currency.ct.toString()}-${currency.id}`;
+
+const minBigNumber = function (n1, n2) {
+    return n1.lt(n2) ? n1 : n2;
+};
```

Comment:
Just to be pedantic -
Either:
    const minBigNumber = (n1, n2) => n1,lt(n2) ? n1 : n2;
or:
    function minBigNumber (n1, n2) {
        return n1,lt(n2) ? n1 : n2;
    }
To mix those two just seem odd :P

Reply:
Agree. Will update. 👍